hc5-async-json/src/main/java/com/ok2c/hc5/json/http/JsonStreamConsumer.java

```diff
@@ -83,7 +86,12 @@ public final void streamEnd(List<? extends Header> trailers) throws HttpExceptio
     }
 
     void failed(Exception cause) {
-        entityConsumerRef.get().failed(cause);
+        AsyncEntityConsumer<T> entityConsumer = entityConsumerRef.get();
+        if (null == entityConsumer) {
+            logger.error("failed before entityConsumer was assigned");
```

Comment:
@scr-oath Is this log entry really useful, especially at the ERROR priority? Could you live without it?  Whenever such condition happens there will always be some sort of transport layer problem which will get logged by HttpClient.

Reply:
@scr-oath I would rather drop the log entry entirely but I can also live with it reduced to DEBUG priority.

Let me know if you agree and I can commit the changeset  and tweak it afterwards.

Reply:
I removed the logger dependency and usage.  Please take another look.